apps/web/server/routers/viewer.tsx

```diff
@@ -373,18 +373,7 @@ const loggedInViewerRouter = createProtectedRouter()
       const orderBy = bookingListingOrderby[bookingListingByStatus];
       const bookingsQuery = await prisma.booking.findMany({
         where: {
-          OR: [
-            {
-              userId: user.id,
-            },
-            {
-              attendees: {
-                some: {
-                  email: user.email,
-                },
-              },
-            },
```

Comment:
On second thought, will this affect team bookings?

Reply:
It depends on who is the organizer because the mail is going to send to the person who does not cancel the event